handler/src/main/java/io/netty/handler/ssl/SslUtils.java

```diff
@@ -107,9 +107,12 @@
     static final String[] TLSV13_CIPHER_SUITES = { "TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384" };
 
     private static final boolean TLSV1_3_SUPPORTED;
+    private static final boolean TLSV1_3_ENABLED;
```

Comment:
Shouldn't those properties named in a way that it's explicit they are specific to the JDK, eg TLSV1_3_JDK_SUPPORTED and TLSV1_3_JDK_SUPPORTED?

Reply:
done